pkg/river/internal/stdlib/stdlib.go

```diff
@@ -15,6 +15,9 @@ import (
 // least one non-error return value, with an optionally supported error return
 // value as the second return value.
 var Functions = map[string]interface{}{
```

Comment:
Maybe it's not a great idea to put non-functions in something called functions. I can rename this or put the constants somewhere else :)

Reply:
Maybe s/Functions/Identifiers? I don't think it's a big deal either way.